src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,52 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 7 * $cell-measure + 6 * $current-gap;
+  gap: $current-gap;
+  padding: 10px;
+
+  &__day {
+    display: none;
+    box-sizing: border-box;
+    width: $cell-measure;
+    height: $cell-measure;
+    background-color: #eee;
+    border: 1px solid #000;
+    cursor: pointer;
```

Comment:
use transition for transform

Reply:
Alexandra, hi! Thanks for help.
I added transaction, but now I can not push this changes to Github( http://joxi.ru/KAxNaWbTVWYyYm

This is the first time when I have problem with "code-pushing" so I am afraid to break something... What should I do?)